### PR TITLE
github: Add mergify configuration

### PR DESCRIPTION
**Pull request by obnoxxx**
config taken from the sambacc config and adjusted

### CODE REVIEW

**Comment by obnoxxx** [comment]
some checks are failing. I assume that this is not caused by this PR and needs to be fixed in master first ...

File: .github/mergify.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  - name: default
+    conditions:
+      - check-success=check
+      - check-success=build-server
+      - check-success=build-ad-server
+      - check-success=build-nightly-server
+      - check-success=build-nightly-adserver
+      - check-success=build-client
+      - check-success=build-toolbox
+      - check-success=test-ad--server-kubernetes
```
Comment (phlogistonjohn) [inline .github/mergify.yml:24]: This double dash doesn't look right to me. But I'm being lazy and haven't double checked.

File: .github/mergify.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  - name: Automatic merge on approval
+    conditions:
+      - check-success=check
+      - check-success=build-server
+      - check-success=build-ad-server
+      - check-success=build-nightly-server
+      - check-success=build-nightly-adserver
+      - check-success=build-client
+      - check-success=build-toolbox
+      - check-success=test-ad--server-kubernetes
```
Comment (phlogistonjohn) [inline .github/mergify.yml:48]: See above.

**Review by phlogistonjohn — CHANGES_REQUESTED** [review]
I'm guessing part of the commit topic got cut off.

`thub:  add mergify configuration`
Should be
`github: add mergify configuration`

File: .github/mergify.yml (hunk 1 above)
Comment (obnoxxx) [reply .github/mergify.yml]: > This double dash doesn't look right to me. But I'm being lazy and haven't double checked.

fixed / removed

**Comment by obnoxxx** [comment]
> I'm guessing part of the commit topic got cut off.
> 
> `thub: add mergify configuration` Should be `github: add mergify configuration`

thanks, you are right.I fixed that.

**Comment by phlogistonjohn** [comment]
> > I'm guessing part of the commit topic got cut off.
> > `thub: add mergify configuration` Should be `github: add mergify configuration`
> 
> thanks, you are right.I fixed that.

Unfortunately, it doesn't appear to have stuck. Github still shows the subject as starting with `thub:`

File: .github/mergify.yml (hunk 2 above)
Comment (obnoxxx) [reply .github/mergify.yml]: > See above.



> See above.


I think I fixed the double dashes yesterday ...

**Comment by anoopcs9** [comment]
> I'm guessing part of the commit topic got cut off.
> 
> `thub: add mergify configuration` Should be `github: add mergify configuration`

Commit message doesn't seem to be fixed yet.

**Comment by phlogistonjohn** [comment]
@obnoxxx  I assume @anoopcs9  and I see the same thing. Here's a screen shot from my browsers view of the commits tab for this PR:

Screenshot_20230125_101051

**Comment by phlogistonjohn** [comment]
A :framed_picture: is :moneybag: :100: :heavy_multiplication_x: 10 :pen:

**Review by phlogistonjohn — APPROVED** [review]
LGTM, thanks!

**Pull request merged**